src/primitives/transaction.h

```diff
@@ -113,13 +103,12 @@ class CPourTx
         return (
             a.vpub_old == b.vpub_old &&
             a.vpub_new == b.vpub_new &&
-            a.scriptPubKey == b.scriptPubKey &&
-            a.scriptSig == b.scriptSig &&
             a.anchor == b.anchor &&
             a.serials == b.serials &&
             a.commitments == b.commitments &&
             a.ciphertexts == b.ciphertexts &&
```

Comment:
(copied and pasted) I'm splitting this kind of stuff into #927 so that I don't need to break serialization again in this PR; we're also not done adding fields and making changes yet anyway.